lib/js-api/src/api/content-rest-api/api/nodes.api.ts

```diff
@@ -982,4 +984,52 @@ export class NodesApi extends BaseApi {
             returnType: DirectAccessUrlEntry
         });
     }
+
+    /**
+     * Initiate a new request to calculate folder size.
+     *
+     * **Note:** this endpoint is available in Alfresco 7.1 and newer versions.
```

Comment:
Wasn't it added in latest version? Does it also have backwards compatibility until 7.1?